Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -131,6 +132,15 @@ open class AVFoundationPlayback: Playback {
     }
 
     open override var position: Double {
+        if let start = dvrWindowStart,
+            let end = dvrWindowEnd,
+            let position = player?.currentItem?.currentTime().seconds {
+            var calculatedPosition = (position - start) * 100
```

Comment:
Is there any reason to have this calculation here? It seems to be unnecessary.

Assuming that `duration == end - start` (at least it should be. The duration reported by the playback should be the duration of the DVR seekable window):
```
                      (position - start) * 100     duration
calculatedPosition = -------------------------- * -----------
                        (end - start) / 100          10000
                      (position - start) * 10000     duration
calculatedPosition = ---------------------------- * ----------
                           (end - start)              10000
                         (position - start)
calculatedPosition = -------------------------- * duration
                           (end - start)
                         (position - start)
calculatedPosition = -------------------------- * duration
                              duration

calculatedPosition = position - start            
```

Reply:
Speaking of the `dvrWindow{Start|End}`, we need to take into account that the seekableTimeRanges could be discontinuous. So merely subtracting could be an issue if there's any discontinuity in the timeline.